tests/unit/Change.spec.ts

```diff
@@ -157,42 +157,6 @@ describe('Change component', () => {
         }
       }))
 
-    // GET business directors
-    get.withArgs('businesses/BC1234567/directors')
```

Comment:
This wasn't used in this scenario. (Change filing fetches parties, not directors.)